src/Showcase/src/Smart.FA.Catalog.Showcase.Infrastructure/Data/Scripts/v_TrainerList.sql

```diff
@@ -1,13 +1,17 @@
 CREATE VIEW [Cfa].[v_TrainerList]
-	AS
-SELECT Cfa.Trainer.Id,
-       Cfa.Trainer.FirstName,
-       Cfa.Trainer.LastName,
-       Cfa.Trainer.Title,
-       Cfa.Trainer.ProfileImagePath,
-       Cfa.Training.TrainingStatusTypeId
-
-FROM Cfa.Trainer
-INNER JOIN Cfa.TrainerAssignment ON Cfa.Trainer.Id = Cfa.TrainerAssignment.TrainerId
-INNER JOIN Cfa.Training ON Cfa.TrainerAssignment.TrainingId = Cfa.Training.Id
-WHERE Cfa.Training.TrainingStatusTypeId = 3
+AS
+SELECT
+    trainer.Id
+    ,trainer.FirstName
+    ,trainer.LastName
+    ,trainer.Title
+    ,trainer.PofileImagePath
+	,count(assignment.TrainingId) As TrainingCount
+FROM Cfa.Trainer trainer
+    INNER JOIN Cfa.TrainerAssignment assignment
+    ON assignment.TrainerId = trainer.Id
+    INNER JOIN Cfa.Training training ON
+    assignment.TrainingId = training.Id
+WHERE training.TrainingStatusTypeId = 3
+GROUP BY trainer.Id, trainer.FirstName, trainer.LastName, trainer.Title, trainer.ProfileImagePath
+HAVING count(assignment.TrainingId) > 1
```

Comment:
Should it not select trainers with at least one training? So rather: 
```suggestion
HAVING count(assignment.TrainingId) >= 1
```